<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTracker</class>
 <widget class="QDialog" name="frmTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmTracker</string>
  </property>
  <widget class="QLabel" name="lblOriginal">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblProcessed">
   <property name="geometry">
    <rect>
     <x>644</x>
     <y>2</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtXYRadius">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>484</y>
     <width>995</width>
     <height>175</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPauseOrResume">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>542</y>
     <width>123</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
